css/styles.css

```diff
@@ -820,6 +863,191 @@ nav .closebtn {
   opacity: 0.96;
 }
 
+.projects-section {
+  display: grid;
+  grid-template-rows: repeat(4, 25%);
+  align-items: center;
+  padding: 115px 25px;
+  gap: 285px;
+  width: 100%;
+  height: 100%;
+  justify-items: center;
+}
+
+/* .tonic-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 25px;
+  gap: 15px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+} */
+
+/** details popup css styles */
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 9999; /* Sit on top */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+
+  /* overflow: auto; Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
+
+/* Modal Content/Box */
+.modal-content {
+  display: inline-block;
+  position: fixed;
+  overflow: auto;
+  left: 50%;
+  top: 1rem;
+  transform: translate(-50%, 0);
+  width: 90%;
+  height: 95vh;
+  border-radius: 16px;
+  padding: 25px;
+  background-color: #fff;

```

Comment:
- [ ] `required`
- [ ] as already mentioned above please make adjustment according to the figma template
- [ ] for all the `mobile pop-up cards`
- [ ]  Please make sure `all` the popups follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., `font`, `colours`, `images`, `text`, `margins`) see screenshot for comparisons

`your template`
![mobile template](https://user-images.githubusercontent.com/66230306/181854539-67fb4811-7f9f-488e-a1fe-0ec7c3416fa2.png)

`figma template`
![mobile pf](https://user-images.githubusercontent.com/66230306/181854597-faff6cc2-9377-47d4-801e-03c0a079fe19.png)

Reply:
@mwanawabangona Could you please check once again the popup I have made in the screenshot you have taken above? If you scroll down, you will the screen coming exactly as it is with all the buttons indicated. Kindly very that.